api/external/infra_proxy/response/environments.proto

```diff
@@ -29,3 +29,8 @@ message Environment {
     // Environment override attributes JSON.
     string override_attributes = 7;
 }
+
+message EnvironmentRecipesList {
+    // Recipes list.
```

Comment:
@vsingh-msys Is this list for _Recipes_ or _Environment Recipes_? This detail should be clarified.

Reply:
It is Environment Recipes list will update it thanks!